nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/TestCSVExportAction.java

```diff
@@ -143,15 +126,21 @@ public void testSimple() throws Exception {
         assertEquals(11, lines.size());
 
         // Check header
-        assertArrayEquals(new String[] { REPOSITORY_FIELD, UID_FIELD, PATH_FIELD, TYPE_FIELD, STATE_FIELD,
-                PARENT_REF_FIELD, IS_CHECKED_OUT_FIELD, IS_VERSION_FIELD, IS_PROXY_FIELD, PROXY_TARGET_ID_FIELD,
-                VERSIONABLE_ID_FIELD, CHANGE_TOKEN_FIELD, IS_TRASHED_FIELD, TITLE_FIELD, VERSION_LABEL_FIELD,
-                LOCK_OWNER_FIELD, LOCK_CREATED_FIELD, LAST_MODIFIED_FIELD }, lines.get(0).split(","));
+        assertArrayEquals(SYSTEM_PROPERTIES_HEADER_FIELDS, lines.get(0).split(","));
+        long count = lines.stream().collect(Collectors.groupingBy(e -> e, Collectors.counting())).get(lines.get(0));
```

Comment:
What do you think about the following, do you think it's more readable ?
```suggestion
        long count = lines.stream().filter(Predicate.isEqual(lines.get(0))).count();
```